examples/platform/policy_service_crud_operations.py

```diff
@@ -71,8 +71,8 @@ def list_policies(cb, parser, args):
     for p in cb.select(Policy):
         print(u"Policy id {0}: {1} {2}".format(p.id, p.name, "({0})".format(p.description) if p.description else ""))
         print("Rules:")
-        for r in p.rules.values():
-            print("  {0}: {1} when {2} {3} is {4}".format(r.get('id'), r.get("action"),
+        for r in p.rules:
+            print("  {0}: {1} when {2} {3} is {4}".format(r.get("id"), r.get("action"),
```

Comment:
Verified fix works.
Could these two lines please be added to src/tests/uat/policy_uat.py at line 31?  Then list will be covered at least with a manual test. 

    # 0. List the policies
    $ python3 examples/platform/policy_service_crud_operations.py --profile PROFILE_NAME --verbose list

Reply:
done